var/spack/repos/builtin/packages/xfsprogs/package.py

```diff
@@ -24,15 +24,15 @@ class Xfsprogs(AutotoolsPackage):
     depends_on("c", type="build")  # generated
 
     depends_on("libinih")
-    depends_on("gettext")
+    depends_on("gettext@:0.21.1")
```

Comment:
If there is a bug report, please link it in a comment. If it is a PR and it is merged, you can already close the xfsprogs range when this restriction applies.

(I got disconnected when posting this the first time. A second comment may appear when my laptop wakes up again.)

Reply:
I didn't find a PR or a bug report, only the Gentoo people having the same issue here: https://bugs.gentoo.org/908864 and this is what made me try with gettext < 0.22.
I don't know if it was fixed in later versions of xfsprog.

Reply:
https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/commit/po/de.po?id=987373623e127fe476d29c81625d288d7a418f95 fixes the issue. This should be in 6.4.0; https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/log/po/de.po?h=v6.4.0

Reply:
Out of scope here, but would be good to update xfsprogs to v6.4.0 or later and avoid pinning gettext.

Reply:
Yeah I tried 6.11.0 but it has other issues (missing dependencies), which would need to be looked into, so I did this PR to fix the version that is in Spack already.

Reply:
Yeah, I think here it would just be sufficient to add the commit above as a comment, and upper limit the restriction (if audit lets you):
```suggestion
    depends_on("gettext@:0.21.1", when="@:6.3")
```

Reply:
Thanks, I committed your suggestion.